pkg/grpc/port.go

```diff
@@ -0,0 +1,17 @@
+package grpc
+
+import "net"
+
+func GetFreePort() (int, error) {
```

Comment:
Nit: It's a public method so it needs comments. linter will not be happy.

Reply:
The linter is happy, but you're right.

Reply:
Updated now.